autopush/endpoint.py

```diff
@@ -728,7 +725,8 @@ def get(self, uaid=""):
         Retrieves the router type/data for a UAID.
 
         """
-        if not self._validate_auth(uaid):
+        test, _ = validate_uaid(uaid)
+        if not test or not self._validate_auth(uaid):
```

Comment:
Perfect. :ok_hand: I saw you used it in the `delete` handler, too, but without `validate_uaid`.